app/upgrades/v15/upgrades.go

```diff
@@ -48,12 +129,32 @@ func CreateV15UpgradeHandler(
 		keepers.TokenFactoryKeeper.SetParams(ctx, updatedTf)
 		logger.Info(fmt.Sprintf("updated tokenfactory params to %v", updatedTf))
 
-		// x/Slashing
-		// Double slashing window due to double blocks per year
-		slashingParams := keepers.SlashingKeeper.GetParams(ctx)
-		slashingParams.SignedBlocksWindow *= 2
-		keepers.SlashingKeeper.SetParams(ctx, slashingParams)
+		// x/Staking - set minimum commission to 0.050000000000000000
+		stakingParams := keepers.StakingKeeper.GetParams(ctx)
+		stakingParams.MinCommissionRate = sdk.NewDecWithPrec(5, 2)
+		err = keepers.StakingKeeper.SetParams(ctx, stakingParams)
+		if err != nil {
+			return nil, err
+		}
 
 		return versionMap, err
 	}
 }
+
+// Previously planned Faster block time upgrade
```

Comment:
move this into the v16 upgrade handler (future) but do not link v16 in the app.go (SDK bug will run this instead of v15 upgrade despite name)